en/flight_modes/flight_modes_summary.md

```diff
@@ -79,22 +79,44 @@ th {
 </tr>
 
 <tr>
- <td><a href="../flight_modes/altitude.md">Altitude</a>
+ <td><a href="../flight_modes/altitude_fw.md">Altitude</a>
  <p><a href="#key_difficulty"><img src="../../assets/site/difficulty_2.svg" title="Difficulty (Easy)" width="20px" /></a></p>
  </td>
  <td><p>S (roll)</p><p>S<sup>+</sup>(pitch)</p></td>
  <td>M</td>
  <td>S<sup>+</sup></td>
  <td><a href="#key_position_fixed"><img src="../../assets/site/position_fixed.svg" title="Altitude required (e.g. Baro, Rangefinder)" width="20px" /></a><sup><a href="#baro_only">++</a></sup></td>
- <td><p>This mode helps vehicle reach and maintain altitude.</p>
+ <td>
+ <!-- 
+ <p>This mode helps vehicle reach and maintain altitude.</p>
  <p>Centered RC RPY sticks gives level flight.</p>
  <p>Pitch input is used to control the altitude. If zero pitch input – autopilot holds current altitude against wind.</p>
  <p>Throttle stick controls the airspeed of the aircraft only if airspeed sensor is connected.</p>
```

Comment:
What if no airspeed sensor is connected? What happens? Does the vehicle just use maximum throttle all the time?